app/scripts/templates/connect_another_device.mustache

```diff
@@ -0,0 +1,52 @@
+<div id="main-content" class="card">
+  <header class="hidden"><h1>{{#t}}Connect another device{{/t}}</h1></header>
+  <div class="success visible">{{#t}}Email confirmed{{/t}}</div>
+  <section>
+    <div class="graphic graphic-connect-another-device">{{#t}}Success{{/t}}</div>
+
+    {{#canSignIn}}
+      <p>
+        {{#t}}Complete set-up by signing in to this Firefox.{{/t}}
+      </p>
+
+      <form novalidate>
+        <div class="button-row">
+          <a id="signin" class="button" href="{{{ escapedSignInUrl }}}">{{#unsafeTranslate}}Sign in &rarr;{{/unsafeTranslate}}</a>
```

Comment:
Very strange, it did for me:

> content-server PORT 3030-3 fxa-content-server.server.requests.INFO: POST /metrics 200 27.231
content-server PORT 3030-3 {"time":"2016-12-15T10:00:00.000Z","op":"client.metrics","hostname":"Shanes-Retina-MBP.local","pid":45211,"v":1,"lang":"en","agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:53.0) Gecko/20100101 Firefox/53.0","duration":4735,"context":"web","broker":"base","entrypoint":"none","service":"none","migration":"none","nt.navigationStart":0,"nt.unloadEventStart":1808,"nt.unloadEventEnd":1808,"nt.fetchStart":1449,"nt.domainLookupStart":1449,"nt.domainLookupEnd":1449,"nt.connectStart":1449,"nt.connectEnd":1449,"nt.requestStart":1450,"nt.responseStart":1806,"nt.responseEnd":1806,"nt.domLoading":6,"nt.domInteractive":28,"nt.domContentLoadedEventStart":29,"nt.domContentLoadedEventEnd":30,"nt.domComplete":1878,"nt.loadEventStart":1878,"nt.loadEventEnd":1878,"events":["experiment.treatment.mailcheck.enrolled","screen.connect-another-device","loaded"],"event_durations":[2627,2643,2646],"screen.width":1680,"screen.height":1050}